scripts/docs_version_pragma_check.sh

```diff
@@ -117,9 +117,10 @@ function findMinimalVersion()
         fi
     done
 
-    if [ -z version ]
+    if [ -z "$version" ]
```

Comment:
But due to this the error was never actually displayed (`-z version` is always true).